search-commons/src/test/java/no/unit/nva/search/OpensearchTest.java

```diff
@@ -114,7 +114,7 @@ void shouldReturnZeroHitsOnEmptyViewingScope() throws Exception {
         indexingClient.addDocumentToIndex(getIndexDocument(indexName, Set.of()));
 
         Thread.sleep(DELAY_AFTER_INDEXING);
-        SearchTicketsQuery searchTicketsQuery = new SearchTicketsQuery(PAGE_SIZE, PAGE_NO);
+        SearchTicketsQuery searchTicketsQuery = new SearchTicketsQuery(PAGE_SIZE, PAGE_NO, null);
```

Comment:
Instantiating an object passing null value in constructor is not recommended. In this case, Collections.emptyList() to indicate no aggregations seems like a better approach to me.

Reply:
Done.